src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -1,14 +1,17 @@
 <%@ tag description="instructorFeedbacks and instructorHome - Feedback Session publish/unpublish button" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
-<%@ attribute name="buttonType" required="true" %>
-<a class="btn ${buttonType} btn-tm-actions session-${publishButton.actionNameLowercase}-for-test"
-   href="${publishButton.actionLink}"
-   title="${publishButton.tooltipText}"
-   data-toggle="tooltip"
-   data-placement="top"
-   data-fsname="${publishButton.feedbackSessionName}"
+<%@ attribute name="showTooltip" type="java.lang.Boolean" required="true" %>
+<%@ attribute name="buttonType" %>
+<a href="${publishButton.actionLink}"
+   class="<c:if test="${not empty buttonType}">${buttonType} </c:if>session-${publishButton.actionNameLowercase}-for-test"
+   <c:if test="${showTooltip}">
+     title="${publishButton.tooltipText}"
+     data-toggle="tooltip"
+     data-placement="top"
+   </c:if>
    data-sending-published-email="${publishButton.sendingPublishedEmail}"
+   data-fsname="${publishButton.feedbackSessionName}"
    <c:if test="${not publishButton.actionAllowed}">disabled</c:if>>
```

Comment:
The `disabled` attribute does not seem to be working. (Try viewing the page as an observer instructor on both master and your branch.) Maybe you could disable the dropdown instead and remove the unnecessary `disabled` here and for the remind buttons.